requirements.txt

```diff
@@ -1,11 +1,10 @@
-mpmath==1.2.1
-numpy==1.19.5
-onnx==1.9.0
-onnxruntime==1.7.0
-packaging==20.9
-protobuf==3.15.8
-pyparsing==2.4.7
-six==1.15.0
-sympy==1.8
-torch==1.10.0
-typing-extensions==3.7.4.3
+numpy>=1.19.5
+onnx>=1.10.2
+onnxruntime>=1.9.0
+pytest>=6.2.5
+PyYAML>=6.0
+setuptools>=59.2.0
+sympy>=1.9
+torch>=1.10.0
```

Comment:
If you change the version here, you should edit corresponding version in `README.md:L8`